saleor/graphql/app/mutations.py

```diff
@@ -220,3 +206,139 @@ def clean_instance(cls, info, instance):
             msg = "You can't delete this app."
             code = AppErrorCode.OUT_OF_SCOPE_APP.value
             raise ValidationError({"id": ValidationError(msg, code=code)})
+
+
+class AppDropFailedInstallation(ModelDeleteMutation):
```

Comment:
To be consistent with the rest of API I would say `AppDeleteFailedInstallation` and replace all "drops" in descriptions below.